Service/ContentTypeService.php

```diff
@@ -438,13 +462,65 @@ public function contentTypeFromJson(string $json, Environment $environment): Con
         return $contentType;
     }
 
+    private function deleteFields(ContentType $contentType): void
+    {
+        $em = $this->doctrine->getManager();
+        $contentType->unsetFieldType();
+        /** @var FieldTypeRepository $fieldRepo */
+        $fieldRepo = $em->getRepository('EMSCoreBundle:FieldType');
+        $fields = $fieldRepo->findBy([
+            'contentType' => $contentType
```

Comment:
```suggestion
            'contentType' => $contentType->getName()
```

It's clearer for the programmer to be explicit on the value that is passed (in stead of relying on the __toString() convention). Also the IDE now knows that there is a dependency between this function and the name of the contentType